mailroom1.py

```diff
@@ -0,0 +1,75 @@
+#Mailroom.py
+
+donors = ["Jeff Bezons","Mark Zuckerberg","Paul Allen","William Gates"]
+donation_amounts = []
+donation_amounts.append([877.33, 10.00,9.00])
+donation_amounts.append([16396.10, 99.25])
+donation_amounts.append([708.42])
+donation_amounts.append([653784.49, 1000.00, 10000.00])
+
+
+
+def thanks():
+    while True:
+        name = input("Please enter your Full Name\n")
+        if name == "list" or name == "List":
+            print(donors)
```

Comment:
you can simple coerce name to lower or uppercase and then compare: 

if name.lower() == "list":